client/src/components/ReviewWizard/Form.tsx

```diff
@@ -43,133 +43,141 @@ export default function Form({ initialReview, onChange, onSubmit }: FormProps):
     reValidateMode: 'onBlur',
   });
 
-  const convertAndSave = () => onChange(convertFormValuesToReview(getValues()));
+  const [lastSavedValues, setLastSavedValues] = useState(getValues());
+
+  const convertAndSave = () => {
+    // only save the draft if the new values are different from the old values
+    const currentValues = getValues();
+    if (!_isEqual(currentValues, lastSavedValues)) {
```

Comment:
lots of ways to do this (like `JSON.stringify`) but I thought `_isEqual` was more expressive in terms of indicating that we care about deep equality